msl_expressions/autogenerated/src/Plans/Attack/DribbleToAttackPointConservative.cpp

```diff
@@ -59,7 +59,10 @@ namespace alica
         {
             send(tmpMC);
         }
-        send(bm);
+        else
```

Comment:
is this an actual bug fix? what about checking translation of bm for NaN? @StephanOpfer @StefanJakob @SirLagsALot23